pkg/kubelet/dockershim/docker_service.go

```diff
@@ -561,7 +562,7 @@ func effectiveHairpinMode(s *NetworkPluginSettings) error {
 			// This is not a valid combination, since promiscuous-bridge only works on kubenet. Users might be using the
 			// default values (from before the hairpin-mode flag existed) and we
 			// should keep the old behavior.
-			klog.Warningf("Hairpin mode set to %q but kubenet is not enabled, falling back to %q", s.HairpinMode, kubeletconfig.HairpinVeth)
+			klog.InfoS("Hairpin mode is set but kubenet is not enabled, falling back hairpin mode", "hairpinMode", s.HairpinMode, "fallBackHairpinMode", kubeletconfig.HairpinVeth)
```

Comment:
this one is tricky.

Reply:
Isn't HairpinVeth just a const? we could just use name of const here. 
```suggestion
			klog.InfoS("Hairpin mode is set but kubenet is not enabled, falling back  to HairpinVeth", "hairpinMode", s.HairpinMode)
```